app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,57 @@
+.row {
+  display: flex;
+  margin-top: 40px;
+}
+
+/* Create two equal columns that sits next to each other */
+.column {
```

Comment:
no need of this you can use bootstrap4 grid system for this (please read grid system, flex of bootstrap4)